### PR TITLE
Update Specviz GUI data loading docs

### PR DESCRIPTION
**Pull request by rosteen**
Mostly updating screenshots to reflect the update UI, simple enough.

### CODE REVIEW

File: docs/specviz/import_data.rst
Comment (pllim) [inline docs/specviz/import_data.rst:29]: Nitpick: Is it possible to break this paragraph into shorter lines so it is easier to review? Thanks!

File: docs/specviz/import_data.rst
Comment (pllim) [inline docs/specviz/import_data.rst:32]: Maybe clarify what one should expect when incompatible spectrum is displayed? Will it crash at loading/selection or something?

**Comment by pllim** [comment]
p.s. I wonder if we want to set `data_label` in the examples and screenshots so the Data dropdown not polluted by randomly generated labels.

p.p.s. I never knew you have to use `load_spectrum`. I always thought it is `load_data`.

**Review by pllim — APPROVED** [review]
Just nitpicks, otherwise LGTM. Thanks!

**Comment by rosteen** [comment]
@pllim On the one hand, I agree that the screenshot would look better with a good data label, but on the other hand you can't specify a label when loading from the GUI, so maybe the screenshot should be representative of what the user will see in that case?

**Comment by pllim** [comment]
> should be representative

Ah, the pesky standalone app. OK, that makes sense.

I already approved, so unless you want a second approval, feel free to merge when CI passes (ignore the dev job failure).

**Pull request merged**
